app/src/main/java/org/schabi/newpipe/util/Localization.java

```diff
@@ -134,7 +135,19 @@ public static String localizeNumber(final Context context, final long number) {
 
     public static String localizeNumber(final Context context, final double number) {
         NumberFormat nf = NumberFormat.getInstance(getAppLocale(context));
-        return nf.format(number);
+        switch ((getAppLocale(context).toString())) {
+            case "zh_CN":
```

Comment:
In Japanese and Korean, numbers are read every 4 digits, but are separated by a comma every 3 digits. So, I'd like to add that it's correct that it's not included in this case classification.